lighthouse-core/lib/network-recorder.js

```diff
@@ -168,15 +165,17 @@ class NetworkRecorder extends EventEmitter {
   }
 
   /**
-   * Construct network records from a log of devtools protocol messages.
-   * @param {!DevtoolsLog} log
+   * Construct network records from a log of devtools protocol messages. If a
+   * driver is provided, it will be used for request content lookups.
+   * @param {!DevtoolsLog} devtoolsLog
+   * @param {!Driver=} driver
```

Comment:
oh let's pick 2, pick 2, pick 2, 😄 , can we @paulirish please? it's a bit silly to try to conform to the interface only here, when it makes the rest of the code more complicated especially when the assumptions of devtools are just very different than the assumptions in LH